src/screening/tbTest/tbTestFilters.js

```diff
@@ -0,0 +1,10 @@
+import { patientObjByEncounterTypeFilter  } from "@openmrs/react-components";
+import { ENCOUNTER_TYPES } from "../../constants";
+
+export default {
+  // doing this puts this on the task list in optional, else it goes to the special list
+  // what condition does a patient has to have determines where this should go to...
+  // feedback ?
+  required: patient => !patient,
```

Comment:
I'm thinking this can just be patient => false to always return false.  (looking at the other filters for nutrition, tb, etc that are always required, I think they should really turned patient => true instead of patient => patient... not sure if there was a particular reason we did it that way.